docs/governance/README.md

```diff
@@ -0,0 +1,129 @@
+
+To ensure that's the case, there is a formal process for proposing and accepting
+APIs into the .NET Standard. This document outlines the process.
+
+## Review board
+
+Today, we have several different .NET implementations:
+
+* .NET Core
+* .NET Framework
+* Mono
+* Unity
+* Xamarin
+
+Some of them have drastically different constraints, based on their runtime
+environment (for instance, Xamarin for iOS cannot perform runtime code
+generation, based on Apple's app store policy).
+
+To ensure we don't end up adding large chunks of API surface that cannot be
+implemented, we have a review board that has to sign-off on API additions to the
+.NET Standard. The board comprises representatives from the following groups:
+
+  * **.NET platform**. The rationale here is that most, if not all, of the APIs
+    that are part of .NET Standard are implemented and evolved by the .NET
+    platform team.
+  * **Xamarin & Mono**. While Mono mostly copies code from the .NET Framework &
+    .NET Core base class libraries, changes and extensions can impact their
+    ability to support the .NET Standard. Thus, we need to coordinate any
+    changes with Mono.
+  * **Unity**. Same rationale as for Xamarin & Mono.
+
+The chairman of the review board is [@migueldeicaza](https://github.com/migueldeicaza).
+For the most part, we strive to make decisions based on consensus, but as it is
+with every group of engineers, consensus can be impossible to achieve at times
+so we need to have an ultimate tie breaker. And Miguel has a lot of expertise
+and experience building .NET implementations that are supported by multiple
+parties.
+
+Please note that the member list isn't meant to be closed: as more platform
+vendors and API drivers appear, the review board will expand accordingly.
+
+## Process
+
+* **Anybody can submit proposals** for API additions to the .NET Standard.
+  Those will be tracked as issues and labeled with the [netstandard-api] label.
+* **We'll track features, no individual APIs**. In order to keep bookkeeping we
+  generally will usually file an issue per feature, rather than per API. The
+  issue description will then contain the APIs that should be added.
+* **New members on standardized types are automatically considered**. To prevent
+  accidental fragmentation, we'll automatically consider all members added by
+  any .NET implementation on types that are already in the standard. The
+  rationale here is that divergence at that the member level is not desirable
+  and unless there is something wrong with the API it's likely a good addition.
+* **Acceptance requires**
+  - A **sponsorship from a board member**. That person will be assigned the
+    issue and is expected to shepherd the issue until it's either accepted or
+    rejected. If no board member is willing to sponsor the proposal, it's
+    considered rejected.
+  - A **stable implementation** in at least one .NET implementation. The
```

Comment:
> in at least **one** .NET implementation

Been thinking about this and discussing it... The timing of what goes into a standard is obviously important, as you want the waves to be "implementable" prior to the next standard and the standards are always additive.

So only one implementation may be problematic as you may have an implementation that has all the features from x.4 but is missing features from x.3 so it can't implement x.3 or x.4 or any later standard (until it has the x.3 features) so prehaps the web browser standardization process should be followed?

i.e. in at least **two** .NET implementations

Don't make an abstraction if you are only going to have one implementation? When you get more than one, then make an abstraction...